lib/rules/forbid-prop-types.js

```diff
@@ -162,7 +162,7 @@ module.exports = {
           checkProperties(node.properties);
           break;
         case 'Identifier': {
-          const propTypesObject = variableUtil.findVariableByName(context, node.name);
+          const propTypesObject = variableUtil.findVariableByName(context, node);
```

Comment:
should this take `node, node.name`? or is that the default behavior

Reply:
This is a temporary (I hope) workaroud for this case:
```ts
      'Program:exit'() {
        jsxElements.forEach((node) => {
          const openingEl = node.openingElement;
          const elName = elementType(openingEl);

          if (fragmentNames.has(elName) || refersToReactFragment(node, elName)) {
//...
const elementType = require('jsx-ast-utils/elementType');
//...
function refersToReactFragment(node, name) {
      const variableInit = variableUtil.findVariableByName(context, node, name)
```